### PR TITLE
Multi molecule handler

### PR DESCRIPTION
**Pull request by HajimeKawahara**
This is a beta version of `multimol` handler. I imported the methods from @ykawashima 's code.  @ykawashima, can you check this PR with files I will send. I want to make this functionality step by step.  The next PR should include more user-friend samples, but this PR, is kind of the pilot version.

### CODE REVIEW

File: src/exojax/utils/molname.py [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+- To get the recommended ExoMol database, use radis.api.exomolapi.get_exomol_database_list("CO2","12C-16O2")
+
+"""
 from radis.db.classes import get_molecule
 import re
 import warnings
 
-
-def exact_hitran_isotope_name_from_isotope(simple_molecule_name, isotope):
+def exact_exact_molecule_name_from_isotope(simple_molecule_name,
```
Comment (HajimeKawahara) [inline src/exojax/utils/molname.py:16]: Double exact

File: src/exojax/spec/multimol.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        "CH4": "CH4/06_HITEMP2020/06_HITEMP2020.par.bz2",
+        "NO": "NO/08_HITEMP2019/08_HITEMP2019.par.bz2",
+        "NO2": "NO2/10_HITEMP2019/10_HITEMP2019.par.bz2",
+        "OH": "OH/13_HITEMP2020/13_HITEMP2020.par.bz2"
+    }
+    return _hitemp_dbpath[simple_molname]
+
+
+def database_path_exomol(simple_molname, root_database_path=".database"):
+    from exojax.utils.molname import simple_molname_to_exact_exomol_stable
```
Comment (HajimeKawahara) [inline src/exojax/spec/multimol.py:209]: put a comment

File: src/exojax/utils/molname.py [hunk 3]
```diff
@@ -1,22 +1,38 @@
+"""molecular name conversion
+
+- CO -> 12C-16O : simple_molname_to_exact_exomol_stable
+- (12C)(16O) -> 12C-16O : exact_molname_hitran_to_exomol
+- (12C)(16O) -> CO : exact_molname_hitran_to_simple_molname
+- 12C-16O -> (12C)(16O) : exact_molname_exomol_to_hitran
+- 12C-16O -> CO exact_molname_exomol_to_simple_molname
+- CO+isotope -> (12C)(16O)  or 12C-16O :exact_exact_molecule_name_from_isotope
```
Comment (HajimeKawahara) [inline src/exojax/utils/molname.py:8]: exact_exact -> exact

**Review by ykawashima — APPROVED** [review]
Thank you very much for the implementation! I have confirmed the changes made and that the sample codes for optimization and hmc you sent work without any problem. I have just added a few suggestions I noticed.

**Pull request merged**